main/guides/getting-started/README.md

```diff
@@ -293,8 +293,15 @@ Once your wallet is connected, click on the "Make Offer" button to purchase 3 pr
 
 ![Making an offer](./assets/new_004.png)
 
-When the transaction is complete you will notice some IST has been debited from your wallet, and you are the owner of three new properties.
+When the transaction is complete you will notice some IST has been debited from your wallet, and you are the owner of three new properties. Feel free to experiment with offering less than the required 0.25 IST or attemping to purchase more than three items. You'll see these offers will not be accepted. Conditional offers like this are a key part of the Agoric platform!
 
 ![Finished transaction](./assets/new_006.png)
 
-Congratulations! You got your first Agoric dapp up and running!
+Congratulations! You got your first Agoric dapp up and running! Now that you've got your first Agoric dapp up and running, let's focus on some key takeaways from this exercise:
+- **Starting an Agoric Chain**: You have gained experience starting a local Agoric blockchain. 
+
+- **Deploying a Contract**: You have deployed a smart contract onto the Agoric platform!
+
+- **Making an Offer**: Finally, you've learned to make an offer and seen that offer constraints are enforced by the Agoric platform. This means user interface can't attempt to fool a user by showing a 0.25 IST when actually charging them a much higer amount, such as 500 IST. It's also important to note that the contract cannot take the 0.25 IST without satisfying the want constraint, nor can the contract take more than the 0.25 cents in the give contraint.
```

Comment:
Oh... we really do need that Keplr offer screenshot to go with this point.

 - https://github.com/Agoric/documentation/issues/921